suite-common/toast-notifications/src/notificationsActions.ts

```diff
@@ -48,6 +54,13 @@ export const addToastOnce = createActionWithExtraDeps(
     },
 );
 
+export const removeToastByType = createAction(
```

Comment:
Name should be removeToast**s**ByType because it will remove all toasts by type.

Also why we need this?